### DISCUSSION TITLE
How to delete files with garbled names that lead to error 404<title>

### DISCUSSION THREAD

**Discussion by guardianVDS** [community/community#102046] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a few files that show up in my repository with garbled names. The names are characters, but not alphabetical ones. When I click on the file name, I get error 404. I need to delete these files from the repository. I am unable to pull the repository down to clean it up as these files prevent the Pull and fetch to work. Does anyone know how I can remove files from the repository on Github website?

**Comment by naveen-98** [comment]
Hey!

If you're facing issues with files having garbled names in your GitHub repository and encountering 404 errors when trying to access them, you can follow these steps to remove them from the repository using the GitHub website:

1. **GitHub Website:**
   - Log in to your GitHub account and navigate to the repository containing the problematic files.

2. **Branch Protection:**
   - Ensure that you are not on a protected branch. Branch protection settings might prevent you from making changes directly on the branch.

3. **Branch Selection:**
   - Select the branch containing the problematic files.

4. **File Navigation:**
   - Navigate to the directory where the garbled files are located.

5. **Delete Files:**
   - Locate the garbled files, click on the file you want to delete, and then click on the "Trash" icon or "Delete" option. This action will mark the file for deletion.

6. **Commit Changes:**
   - Scroll down to the bottom of the page to find the commit changes section. Provide a commit message explaining the deletion and commit the changes.

7. **Create Pull Request (Optional):**
   - If you are working with branches, you might need to create a pull request to merge the changes into the main branch. Follow the instructions on the GitHub website to create and merge the pull request.

8. **Cleanup Local Repository (Optional):**
   - After making changes on the GitHub website, you can then clean up your local repository by pulling the changes. Use the following command in your local terminal:

      ```bash
      git pull origin branch-name
      ```

   Replace "branch-name" with the name of your branch.

9. **Verify Changes:**
   - Verify that the garbled files are no longer present in the repository by checking the repository on the GitHub website.

Remember to be cautious while deleting files, and ensure that you are not deleting critical files by mistake.

**Comment by iamaloksy** [comment]
Response #2

**Comment by guardianVDS** [comment]
Naveen

Hi. Thanks for the response and the steps. I have tried to do all of this. The issue is that when I go to the directory and click on the garbled file, I get a screen with error 404. It says that the page not found:



It says that:

The master branch does not contain the file.

So, I cannot delete it or do anything else.
